pkg/baseline/packageIndex.json

```diff
@@ -319,9 +319,10 @@
       "StableVersions": [
         "1.6.0",
         "1.6.1",
-        "2.0.0"
+        "2.0.0",
+        "2.0.1"
       ],
-      "BaselineVersion": "2.0.0",
+      "BaselineVersion": "2.0.1",
```

Comment:
Not sure we want to update the baseline here but also don't know that it really affects anything because nothing consumes it.

Reply:
I see, I thought it was required, I can remove that change if we don't need it for marking this stable